packages/next-server/server/next-server.ts

```diff
@@ -38,6 +38,7 @@ export default class Server {
     buildId: string,
     generateEtags: boolean,
     runtimeConfig?: {[key: string]: any},
+    crossOrigin?: string,
```

Comment:
Instead of passing it all the way through the app (and increasing html size) let's instead use `webpack.DefinePlugin` to inline the value, this also makes the changes needed less code and more reliable in the future 😌

Reply:
Very true... didn't think about that :+1:

Reply:
Really good way of thinking: Compile time is better than runtime

Reply:
I was just worried about breaking dynamic asset paths as if someone might be able to change crossOrigin on demand but I don't think there's really a case for it.

Reply:
Yeah agreed, basically keeping this issue in mind: https://github.com/zeit/next.js/issues/5846